mitodl/ocw-studio
# Updating pipeline definitions to point at ocw-course-publisher v0.4

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-studio/issues/1645.

#### What's this PR do?
Updates the pipeline definitions to point at the new version of ocw-course-publisher (v0.4).

#### How should this be manually tested?
Analogously to testing for https://github.com/mitodl/ocw-studio/pull/1321, test via the following steps:
1. Make sure that your `.env` file contains
```
CONCOURSE_URL=http://concourse:8080
CONCOURSE_PASSWORD=test
CONCOURSE_USERNAME=test
CONCOURSE_TEAM=main
```
and that your `/etc/hosts` file contains the line `127.0.0.1 concourse`.

2. Start OCW Studio with `docker compose up`.
3. Upsert a pipeline with `docker compose exec web ./manage.py backpopulate_pipelines --filter <site name>`
4. Inspect the pipeline definition with `fly -t local get-pipeline -p live/site:<site name>` and make sure that `ocw-course-publisher` has version 0.4 specified. To get the pipeline name, you can first run `fly -t local pipelines | grep <part of course name>`.
